scripts/build

```diff
@@ -6,9 +6,16 @@ source $(dirname $0)/version
 cd $(dirname $0)/..
 
 mkdir -p bin
+
 [ "$(uname)" != "Darwin" ] && LINKFLAGS="-extldflags -static -s"
+
+if [ "$(git rev-parse --abbrev-ref HEAD)" = "master" ]; then
```

Comment:
nit: we should support different branches as well including master/v[1-9].[1-9].x.

Reply:
Update in https://github.com/longhorn/longhorn-manager/pull/2298